examples/gallery/lines/roads.py

```diff
@@ -14,8 +14,10 @@
 # %%
 import geopandas as gpd
 import pygmt
+import pyogrio
 
 # Read shapefile data using geopandas
+pyogrio.core._register_drivers()
 gdf = gpd.read_file(
```

Comment:
Found this issue https://github.com/geopandas/pyogrio/issues/144 that mentions an error like `pyogrio.errors.DataSourceError: Could not obtain driver: GPKG (check that it was installed correctly into GDAL)` was due to drivers not being loaded properly. This should have been fixed in https://github.com/geopandas/pyogrio/pull/145, but unsure why we're still hitting into this in the sphinx build and not in the regular tests.